recipes/xproperty/all/conanfile.py

```diff
@@ -12,25 +14,56 @@ class XpropertyConan(ConanFile):
     name = "xproperty"
     description = "Traitlets-like C++ properties and implementation of the observer pattern."
     license = "BSD-3-Clause"
-    topics = ("observer", "traitlets")
-    homepage = "https://github.com/jupyter-xeus/xproperty"
     url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://github.com/jupyter-xeus/xproperty"
+    topics = ("observer", "traitlets", "header-only")
     package_type = "header-library"
     settings = "os", "arch", "compiler", "build_type"
     no_copy_source = True
 
+    @property
+    def _min_cppstd(self):
+        return "14" if Version(self.version) <= "0.12.0" else "17"
```

Comment:
This error is a bit suprising, as per [jupyter-xeus/xproperty@`0.11.0...0.12.0`#diff-33394812ba](https://github.com/jupyter-xeus/xproperty/compare/0.11.0...0.12.0#diff-33394812ba204689144fd2f80832db83853ba1cb32403edb4e15fe4893e675fd)L40 it seems like they expect to be able to use C++14? Did you get a chance to communicate this to upstream?

Reply:
@RubenRBS 
Thanks for your comment!
I create the issue about it. https://github.com/jupyter-xeus/xproperty/issues/69